MonoGame.Framework/SDL/SDLGamePlatform.cs

```diff
@@ -39,17 +39,24 @@ public SdlGamePlatform(Game game)
             Sdl.Minor = sversion.Minor;
             Sdl.Patch = sversion.Patch;
 
+            try
+            {
+                Sdl.Window.SetResizable(IntPtr.Zero, false);
+                Sdl.Patch = 5;
```

Comment:
@cra0zy can we add a comment here that this is a "hack" to work around people compiling SDL2 themselves and not updating the version number. Ideally this will be removed at some point once we get a release with the fixes in.